Sources/RingPublishingTracking/Public/RingPublishingTracking.swift

```diff
@@ -116,7 +116,10 @@ public class RingPublishingTracking {
     ///
     /// - Parameter events: [Event]
     public func reportEvents(_ events: [Event]) {
-        Logger.log("Reporting generic events, events count: '\(events.count)'")
```

Comment:
a potem nie ma logów z tym? bo to jest kolejkowane potencjalnie tez

Reply:
Zamieniłem to tylko, że wyświetla w pętli log per event.
Nie rozumiem co rozumiesz przez "jest kolejkowane"

Reply:
nvm :)